### PR TITLE
SIDE-2062 Add self test loop code to javascript LDK

### PR DESCRIPTION
**Pull request by BrettMugglinOlive**

### CODE REVIEW

File: ldk/javascript/example/self-test-loop/generate-banner.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function generateMetadata() {
```
Comment (RusGraf) [inline ldk/javascript/example/self-test-loop/generate-banner.js:1]: I thought, this banner should be in the javascript ldk and not in the loop.

File: ldk/javascript/example/self-test-loop/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "loop-goja",
```
Comment (RusGraf) [inline ldk/javascript/example/self-test-loop/package.json:2]: Should we update the name to something close to the self-test-loop.

File: ldk/javascript/example/self-test-loop/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+const webpack = require("webpack");
```
Comment (RusGraf) [inline ldk/javascript/example/self-test-loop/webpack.config.js:1]: I think this is also part of the LDK.

File: ldk/javascript/example/self-test-loop/generate-banner.js (hunk 1 above)
Comment (BrettMugglinOlive) [reply ldk/javascript/example/self-test-loop/generate-banner.js]: That's been removed at this point

File: ldk/javascript/examples/self-test-loop/src/selfTest.ts [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+      console.log('Error Streaming', 'error', e.toString());
+    }
+  }
+
+  openTestGroups(): void {
+    let allTests = [] as LoopTest[];
+    // eslint-disable-next-line
+    const clickableElements: any[] = [];
+    // eslint-disable-next-line
+    for (const config in testConfig) {
```
Comment (BrettMugglinOlive) [inline ldk/javascript/examples/self-test-loop/src/selfTest.ts:240]: Remove this

**Review by cshields-oliveai — APPROVED** [review]
Thanks for the overview. Looks good. Should we create new tasks for continuing to add tests for aptitudes?

**Review by RusGraf — APPROVED** [review]

**Pull request merged**